src/OSPSuite.Presentation/Presenters/SimulationOutputMappingPresenter.cs

```diff
@@ -191,5 +191,21 @@ public void Handle(SimulationOutputSelectionsChangedEvent eventToHandle)
          if (Equals(eventToHandle.Simulation, _simulation))
             Refresh();
       }
+
+      public void RemoveMultipleObservedData(IEnumerable<SimulationOutputMappingDTO> lstOutputMappingDTO)
+      {
+
+         var usedObservedDataList = lstOutputMappingDTO
+            .Select(outputMappingDTO => new UsedObservedData
+            {
+               Id = outputMappingDTO.ObservedData.Id,
+               Simulation = _simulation
+            })
+            .ToList();
+
+
+         _observedDataTask.RemoveUsedObservedDataFromSimulation(usedObservedDataList);
+         MarkSimulationAsChanged();
+      }
```

Comment:
This is too similar to the singular entry `RemoveObservedData` to keep both. Just use this one and always make the caller create an array with single item when needed.